docs/tutorial/force_calibration.rst

```diff
@@ -212,3 +212,98 @@ If we compare the different fits, we can see that the Lorentzian model with diod
+If we choose the windowing to include a full number of periods of the driving input, then the resulting driving peak will end up in a single data point in the frequency spectrum.
+
+.. image:: driving_input.png
+
+The physical spectrum is then given by a thermal part (like before) and an active part:
+
+.. math::
+
+    P_{thermal}(f) = \frac{D}{\pi ^ 2 \left(f^2 + f_c ^ 2\right)}
+
+    P_{active}(f) = \frac{A^2}{2\left(1 + \frac{f_c^2}{f_{drive}^2}\right)} \delta(f - f_{drive})
+
+    P(f) = P_{thermal}(f) + P_{active}(f)
+
+Since we know the driving amplitude, we know how the bead reacts to the driving motion and we can observe this response in the PSD, we can use this relation to determine the positional calibration.
+
+If we use the basic Lorentzian model, then the theoretical power (integral over the delta spike) corresponding to the driving input is given by :cite:`tolic2006calibration`:
+
+.. math::
+
+    W_{theory} = \frac{A^2}{2\left(1 + \frac{f_c^2}{f_{drive}^2}\right)}
+
+Subtracting the thermal part of the spectrum, we can determine the same quantity experimentally.
+
+.. math::
+
+    W_{experiment} = \left(P^{volt}(f_{drive}) - P^{volt}_{thermal}(f_{drive})\right) \Delta f
+
+Here the thermal contribution that needs to be subtracted is obtained from fitting the thermal part of the spectrum using the passive calibration procedure from before.
+The desired positional calibration is then:
+
+.. math::
+
+    \beta = \sqrt{\frac{W_{theory}}{W_{experiment}}}
+
+Note how this time around, we did not rely on assumptions on the viscosity of the medium or the bead size.
+
+As a side effect of this calibration, we actually obtain an experimental estimate of the drag coefficient:
+
+.. math::
+
+    \gamma_{measured} = \frac{k_B T}{\beta^2 D^{volt}}
+
+Using `Pylake`, the procedure to use active calibration is not very different from passive calibration.
+However, it does require some additional data channels as inputs.
+Instead of using the :class:`~.PassiveCalibrationModel` presented in the previous section, we now use the :class:`ActiveCalibrationModel`.
+
+In this tutorial, we're going to assume the nanostage was used as driving input::
+
+    driving_data = f["Nanostage Position"]["Nanostage X"][start:stop]
+
+We also need to provide the sample rate at which the data was acquired, and a rough guess for the driving frequency.
+Pylake will find an accurate estimate of the driving frequency based on this initial estimate (provided that it is close enough)::
+
+    active_model = lk.ActiveCalibrationModel(driving_data.data, volts, driving_data.sample_rate, bead_diameter, driving_frequency_guess=37)
+
+To check the determined frequency, we can look at the determined driving frequency::
+
+    >>> active_model.driving_frequency
+    36.95
+
+We can now use this model to fit the power spectrum::
+
+    calibration = lk.fit_power_spectrum(power_spectrum, active_model)
+    calibration
+
```

Comment:
Nice! I like how this complex computation now looks so simple.

Reply:
That's the idea! :)

Hopefully we can keep it that way as we get to more and more complex methods :D